README.md

```diff
@@ -22,7 +22,7 @@ $ npm i generate-changelog -g # install it globally
 To use this module, your commit messages have to be in this format:
 
 ```
-type(category): description
+type(category): description [flags]
 ```
 
 Where `type` is one of the following:
```

Comment:
Since we're also adding `breaking` as a `type`, can you add it to this list as well?

Reply:
👍